src/pages/projects/ProjectGeneralInfo.tsx

```diff
@@ -96,7 +96,7 @@ export const ProjectGeneralInfo = ({ project }: ProjectGeneralInfoProps) => {
             {project.projectCategories.map((category, index) => (
               <Chip
                 component="dd"
-                sx={{ m: '0' }}
+                sx={{ m: '0', borderColor: 'secondary.main' }}
                 key={index}
                 color="primary"
```

Comment:
var ikke tanken å bruke color og variant i stedet for styling via sx nå?
```suggestion
                sx={{ m: '0'' }}
                key={index}
                variant="outlined"
                color="secondary"
```

Ser ut som eneste forskjellen er om teksten vises som lilla eller svart, men tenker det også burde vært gitt av theme, så det blir likt for alle chips